pkg/config/install_requirements.go

```diff
@@ -316,6 +316,14 @@ type StorageConfig struct {
 	Backup StorageEntryConfig `json:"backup"`
 }
 
+// AWSConfig contains AWS specific requirements
+type AWSConfig struct {
+	// AccountID the unique 12-digit number assigned to your AWS account
+	AccountID string `json:"accountId,omitempty"`
+	// UseIRSAAnnotations if true, IAM roles will be assumed to exist and they'll be linked to ServiceAccounts using annotations
+	UseIRSAAnnotations bool `json:"useIRSAAnnotations"`
```

Comment:
You could try to add `omitempty` here as well.

Reply:
I hadn't noticed that for the other `bool` fields. I thought perhaps it was not needed